explainer.md

```diff
@@ -52,17 +53,22 @@ goals of the project are, and a brief overview of how the solution works.
 This should be no more than 1-2 paragraphs.]
 
 
-## Goals [or Motivating use cases, or Scenarios]
-
-[What is the end-user need which this project aims to address?]
-
+## Goals
+- Allow web applications to evict partitions of data
+- Allow web developers to specify eviction ordering
+- Allow web developers to remove a service workers without clearing data for the entire origin
+- Allow web developers to reserve quota
+- Allow web developers to express performance, durability and other trade-off decisions on partitions of data
+- Ensure privacy across accounts on shared devices
```

Comment:
This goal seems to be the only one that is actually phrased as a "end-user need which this project aims to address". Not sure how to best phrase the end-user need for the rest, but probably something around making sure browsers don't delete data that users don't want to be deleted? (and I would definitely also keep what you have now; but also include what end-user needs we achieve by letting web developers/applications do these things).

Reply:
Good point, I tried adding an additional point about preventing important data from being deleted. Let me know if it doesn't match what you had in mind. Thanks!